lms/djangoapps/mobile_api/testutils.py

```diff
@@ -123,8 +146,228 @@ def test_other_user(self):
         self.api_response(expected_response_code=404, username=other.username)
 
 
+class MobileAPIMilestonesMixin(object):
+    """
+    Tests the Mobile API decorators for milestones.
+
+    The two milestones supported in these tests are entrance exams and
+    pre-requisite courses. If either of these milestones are unfulfilled,
+    the mobile api will appropriately block content until the milestone is
+    fulfilled.
+    """
+    MILESTONE_ERROR = {'developer_message': 'Cannot access content with unfulfilled pre-requisites or unpassed entrance exam.'}  # pylint: disable=line-too-long
+
+    # Enrollment list hides enrolled courses, milestone courses are not hidden
+    ALLOW_ACCESS_TO_MILESTONE_COURSE = False  # pylint: disable=invalid-name
+
+    def __init__(self):
+        self.entrance_exam = None
+        self.problem_1 = None
+        self.milestone = None
+        self.milestone_relationship_types = None
+        self.request = None
+
+    def init_course_access(self, course_id=None):
+        """Base implementation of initializing the user for each test."""
+        self.login_and_enroll(course_id)
```

Comment:
This is the same as the base implementation.  You should remove this `init_course_access` method.